guides/common/modules/proc_assigning-resource-quotas-to-a-user.adoc

```diff
@@ -11,3 +12,29 @@ You can assign resource quotas to a user to limit the resource consumption of th
 +
 This means that {Project} will not prevent users from consuming more resources than their assigned quota.
 . Click *Submit* to assign resource quotas to a user.
+
+[id="cli-assigning-resource-quotas-to-a-user"]
+.CLI procedure
+. Optional: List all {Project} users:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer user list --fields id,login
+----
+. Optional: List all resource quotas:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer resource-quota list --fields id,name
+----
+. Assign resource quotas to a user:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer user update \
+--id _My_User_ID_ \
+--resource-quota-ids _My_Resource_Quota_IDs_ \
+--resource-quota-is-optional _{true|false}_
+----
++
+For all options, see `hammer user --help`.
```

Comment:
You only see the resource quota option for user `create` and `update` commands.

Reply:
Added "update".